shuup/simple_supplier/module.py

```diff
@@ -20,8 +20,11 @@
 class SimpleSupplierModule(BaseSupplierModule):
     identifier = "simple_supplier"
     name = "Simple Supplier"
+    handles_internal_type = [0, 1]
```

Comment:
why 0 and 1?

Reply:
As plan products should be handle by simple supplier but we don't want it to show up in the product list view

Reply:
where is 1 defined?